sourir.ml

```diff
@@ -13,6 +15,8 @@ let () =
     ("--quiet", Arg.Set quiet, "quiet");
     ("--autofix", Arg.Set autofix, "automatically normalize graph");
     ("--opt", Arg.String (fun s -> opts := String.split_on_char ',' s), "Enable optimizations");
+    ("--nopt", Arg.String (fun s -> opt_runs := int_of_string s), "Number of optimization runs");
```

Comment:
Could you rename this to `opt-runs` or `num-opts`? `nopt` looks too much like `nop` or `no opt` to me.